pyke/lightcurve.py

```diff
@@ -150,6 +179,101 @@ def detrend(self):
         """
         pass
 
+class SystematicsCorrector(object):
+    def correct(self):
+        pass
+
+
+class KeplerCBVCorrector(SystematicsCorrector):
+    """Remove systematic trends Kepler light curves by fitting
+    cotrending basis vectors.
+
+    .. math::
+
+         \arg \min_{\theta \in \Theta} |f(t) - <\theta, \left[\rm{cbv}_1(t), ..., \rm{cbv}_n(t)\right]^{T}|^p
+
+    Attributes
+    ----------
+    lc_file : KeplerLightCurveFile object or str
+    cbvs : list of ints
```

Comment:
You may want to add a few more words to explain that it refers to "VECTOR_1", etc...